src/dfmpreview/.openpublishing.build.ps1

```diff
@@ -0,0 +1,17 @@
+param(
```

Comment:
>param [](start = 0, length = 5)

no need to add ps1 and publish.config.json which is for publishing to docs/msdn?